libs/core/src/global/styles/tokens/core/_core.scss

```diff
@@ -21,47 +21,91 @@
   --pine-box-shadow-lg: 0 8px 40px 0 rgba(0,0,0,0.24);
   --pine-color-white: #ffffff;
   --pine-color-black: #000000;
-  --pine-color-grey-100: #f9fafa;
-  --pine-color-grey-200: #f6f8f8;
-  --pine-color-grey-300: #eceeef;
-  --pine-color-grey-400: #d3d5d9;
-  --pine-color-grey-500: #b5bac0;
-  --pine-color-charcoal-100: #8d939a;
-  --pine-color-charcoal-200: #60666c;
-  --pine-color-charcoal-300: #43474b;
-  --pine-color-charcoal-400: #202327;
-  --pine-color-charcoal-500: #040506;
-  --pine-color-blue-100: #e6f4fe;
-  --pine-color-blue-200: #8ecafb;
-  --pine-color-blue-300: #0072ef;
-  --pine-color-blue-400: #054fb8;
-  --pine-color-blue-500: #07265f;
-  --pine-color-green-100: #ddf8f0;
-  --pine-color-green-200: #86d5bc;
-  --pine-color-green-300: #23856d;
-  --pine-color-green-400: #225d53;
-  --pine-color-green-500: #183e3b;
-  --pine-color-red-100: #fff0f0;
-  --pine-color-red-200: #fdb0aa;
-  --pine-color-red-300: #cf3c32;
-  --pine-color-red-400: #99221e;
-  --pine-color-red-500: #5e0d0d;
-  --pine-color-yellow-100: #fef8e1;
-  --pine-color-yellow-200: #fad980;
-  --pine-color-yellow-300: #ffc505;
-  --pine-color-yellow-400: #c56a02;
-  --pine-color-yellow-500: #8c3b08;
-  --pine-color-orange-100: #fef1e1;
-  --pine-color-orange-200: #ffb36b;
-  --pine-color-orange-300: #fb7e09;
-  --pine-color-orange-400: #b3501e;
-  --pine-color-orange-500: #5a260c;
-  --pine-color-purple-100: #eee4f6;
-  --pine-color-purple-200: #d9c2ef;
-  --pine-color-purple-300: #8e5ad8;
-  --pine-color-purple-400: #50348a;
-  --pine-color-purple-500: #381c5e;
-  --pine-font-family-circular: "Circular", -apple-system, "BlinkMacSystemFont", '"Segoe UI"', "Roboto", '"Helvetica Neue"', "Arial", '"Noto Sans"', sans-serif;
+  --pine-color-gray-50: #fcfcfc;
```

Comment:
after today's critique meeting, we've agreed to stay with the `grey` naming for now.